com.unity.render-pipelines.universal/Runtime/Debug/DebugDisplaySettingsCommon.cs

```diff
@@ -0,0 +1,97 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace UnityEngine.Rendering.Universal
+{
+    public class DebugDisplaySettingsCommon : IDebugDisplaySettingsData
```

Comment:
Is this meant to be public? if so needs public API docs.

Reply:
Shouldn't be public, removing.